packages/datadog-plugin-pg/src/index.js

```diff
@@ -9,6 +9,7 @@ class PGPlugin extends DatabasePlugin {
 
   start ({ params = {}, query, processId }) {
     const service = getServiceName(this.config, params)
```

Comment:
This seems pretty weird to me.

Currently we take `this.config.service` and check if it's a function. If so we run the function with the connection as an argument. Otherwise we use the string value as is. The result is then used for the `startSpan()` code and then it is discarded.

Shouldn't we instead replace the `this.config.service` value with the evaluated version?